scripts/inf-create.sh

```diff
@@ -60,6 +60,20 @@ then
     az account set -s "$ARM_SUBSCRIPTION_ID"
 fi
 
+
+# prepare vars for the users you wish to assign to the security group
+object_ids=()
+# Remove spaces from the comma-separated string
+ENTRA_OWNERS=$(echo "$ENTRA_OWNERS" | tr -d ' ')
+IFS=',' read -ra ADDR <<< "$ENTRA_OWNERS"
+for user_principal_name in "${ADDR[@]}"; do
```

Comment:
What happens if an invalid email address is added?

Reply:
This now accomodates a bad email address, so it isn't added to the array of id's passed to TF. It also handles spaces between commas/email addresses